azure-mgmt-graph-rbac/src/main/java/com/microsoft/azure/management/graphrbac/Applications.java

```diff
@@ -0,0 +1,34 @@
+/**
+ * Copyright (c) Microsoft Corporation. All rights reserved.
+ * Licensed under the MIT License. See License.txt in the project root for
+ * license information.
+ */
+
+package com.microsoft.azure.management.graphrbac;
+
+import com.microsoft.azure.management.apigeneration.Beta;
+import com.microsoft.azure.management.apigeneration.Fluent;
+import com.microsoft.azure.management.graphrbac.implementation.ApplicationsInner;
+import com.microsoft.azure.management.graphrbac.implementation.GraphRbacManager;
+import com.microsoft.azure.management.resources.fluentcore.arm.collection.SupportsGettingById;
+import com.microsoft.azure.management.resources.fluentcore.arm.collection.SupportsGettingByNameAsync;
+import com.microsoft.azure.management.resources.fluentcore.arm.models.HasManager;
+import com.microsoft.azure.management.resources.fluentcore.collection.SupportsCreating;
+import com.microsoft.azure.management.resources.fluentcore.collection.SupportsDeletingById;
+import com.microsoft.azure.management.resources.fluentcore.collection.SupportsListing;
+import com.microsoft.azure.management.resources.fluentcore.model.HasInner;
+
+/**
+ * Entry point to application management API.
+ */
+@Fluent(ContainerName = "/Microsoft.Azure.Management.Fluent.Graph.RBAC")
+@Beta
+public interface Applications extends
+        SupportsListing<Application>,
+        SupportsGettingById<Application>,
+        SupportsGettingByNameAsync<Application>,
+        SupportsCreating<Application.DefinitionStages.Blank>,
+        SupportsDeletingById,
+        HasManager<GraphRbacManager>,
+        HasInner<ApplicationsInner> {
+}
```

Comment:
lately, we've also been adding `SupportsBatchDeletion` to collection interfaces. Is it possible to support it here?  See VirtualMachines for reference.